go/test/endtoend/sharding/base_sharding.go

```diff
@@ -0,0 +1,353 @@
+/*
+Copyright 2019 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+package sharding
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"math"
+	"net/http"
+	"reflect"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"vitess.io/vitess/go/test/endtoend/cluster"
+	querypb "vitess.io/vitess/go/vt/proto/query"
+	"vitess.io/vitess/go/vt/proto/topodata"
+)
+
+var (
+	lotRange1 uint64 = 0xA000000000000000
+	lotRange2 uint64 = 0xE000000000000000
+	// InsertTabletTemplateKsID common insert format to be used for different tests
+	//
+	InsertTabletTemplateKsID = `insert into %s (id, msg) values (%d, '%s') /* vtgate:: keyspace_id:%016X */ /* id:%d */`
```

Comment:
Is this still being used? If yes, remove the comments inside the query, those are v2.

Reply:
Since we are inserting to tablets directly, we need this - [link to discussion](https://planetscale.slack.com/archives/GNKDZJZ7T/p1575186830032900).

Reply:
no, you don't need these comments. they are only parsed and used while inserting via vtgate, and even there with v3 you should not need them.